handbook/business-operations/README.md

```diff
@@ -488,25 +488,29 @@ Here are the steps hiring managers follow to get an offer out to a candidate:
 2. **Prepare the "exit scenarios" spreadsheet:** Head of Business Operations [copies the "Exit scenarios (template)"](https://docs.google.com/spreadsheets/d/1k2TzsFYR0QxlD-KGPxuhuvvlJMrCvLPo2z8s8oGChT0/copy) for the candidate, and renames the copy to e.g. "Exit scenarios for Jane Doe".
-   - _Edit the candidate's copy of the exit scenarios spreadsheet_ to include the number of shares they will be offered, and the spreadsheet will update automatically to reflect their approximate ownership percentage.
-   - _Share the candidate's copy_ of the spreadsheet with their personal email, granting **"Edit"** access.
+   - _Edit the candidate's copy of the exit scenarios spreadsheet_ to number of shares offered in ["🥧 Equity plan"](https://docs.google.com/spreadsheets/d/1_GJlqnWWIQBiZFOoyl9YbTr72bg5qdSSp4O3kuKm1Jc/edit#gid=0), and the spreadsheet will update automatically to reflect their approximate ownership percentage.
      > _**Note:** Don't play with numbers in the exit scenarios spreadsheet. The revision history is visible to the candidate, and they might misunderstand._
 3. **Prepare offer email:** Head of Business Operations [copies "Offer email (template)"](https://docs.google.com/document/d/1zpNN2LWzAj-dVBC8iOg9jLurNlSe7XWKU69j7ntWtbY/copy) and renames to e.g. "Offer email for Jane Doe".  Edit the candidate's copy of the offer email template doc and fill in the missing information:
    - _Benefits:_ If candidate will work outside the US, [change the "Benefits" bullet](https://docs.google.com/document/d/1zpNN2LWzAj-dVBC8iOg9jLurNlSe7XWKU69j7ntWtbY/edit) to reflect what will be included through Fleet's international payroll provider, depending on the candidate's location.
-   - _Equity:_ Specify the number of shares, then highlight the number of shares with a link to the candidate's personalized copy of the "Exit scenarios" spreadsheet.
-   - _Other information_: Read the offer email very carefully and double-check correctness versus the equity plan, especially in regards to salary, equity, and start date.
-4. **Send offer:** Once both documents are complete, Head of Business Operations notifies Apprentice to the CEO who then drafts the offer email, bringing it to the CEO to send during their daily meeting:
+   - _Equity:_ Highlight the number of shares with a link to the candidate's custom "exit scenarios" spreadsheet.
+4. **Review offer email doc:**
+   - Read the offer email very carefully and double-check correctness versus the equity plan, especially in regards to salary, equity, and start date.
+   - Check all links in offer letter for accuracy, clicking surrounding areas to ensure no "ghost links" are left from previous edits. **Note:** This has happened before.
+Once both documents are complete, Head of Business Operations shares the "offer email doc" with the [Apprentice to the CEO](https://fleetdm.com/handbook/company/ceo#team) 
+5. **Draft email in CEO's Gmail**_To be completed by the Apprentice to the CEO_ 
```

Comment:
Step 4 is draft and review.  Step 5 is literally hit send and then also share exit scenarios